### ISSUE TITLE
__version__ attribute is unavailable

### ISSUE THREAD

**Issue by maxalbert** [lelit/pglast#12]
Minor issue, but the `pglast` module does not have a `__version__` attribute:
```
>>> import pglast
>>> print(pglast.__version__)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-8eeeaab38785> in <module>
      1 import pglast
----> 2 print(pglast.__version__)

AttributeError: module 'pglast' has no attribute '__version__'
```
It would be helpful to add this in order to be able to check the version from within Python.

**Comment by lelit** [comment]
I will add that sooner or later.

**Comment by maxalbert** [comment]
Great, thanks! For what it's worth, I've found [versioneer](https://github.com/warner/python-versioneer) quite helpful for sensible, effectively automatic version management.

**Comment by lelit** [comment]
Do you have particular needs, or would it be enough having `__version__` carrying the same value as the release tag, i.e. something like `v1.2`?

**Comment by maxalbert** [comment]
As a user, that would certainly be sufficient because I'm primarily interested in whether I am running the latest released version or an old one.

**Comment by lelit** [comment]
This has been released as of version 1.3. Thank you!

**Comment by maxalbert** [comment]
Great, thanks!
